README.md

```diff
@@ -1 +1,19 @@
 # Firefox profiler server
+[![Matrix][matrix-badge]][matrix]
+
+Welcome to the repository for the Firefox profiler server! This server
+implements some APIs needed by the profiler UI.
+
+Here are some pointers to know more:
```

Comment:
This sentence is a little awkward sounding to my ears.

Maybe:

> Here are some links to learn more:

"Pointers" are more directly actionable tips.